src/content/docs/contributing/documentation/docs_guidelines.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Writing style
+subtitle: Guidelines for contributing to nf-core documentation.
+weight: 10
+parentWeight: 40
+markdownPlugin: addNumbersToHeadings
+---
+
+# Documentation contribution guidelines
+
+Many thanks for taking an interest in improving the nf-core documentation. To maintain consistency and clarity across our documentation, please follow these contribution guidelines.
+
+## General writing guidelines
+
+### Use British English
+
+Write in British English, not US English.
+
+- ✅ _"colour"_ instead of ❌ _"color"_
+- ✅ _"organise"_ instead of ❌ _"organize"_
+
+### Use active voice
+
+Active voice makes sentences clearer and more direct.
+
+- ✅ Active: _"nf-core curates pipelines."_
+- ❌ Passive: _"Pipelines are curated by nf-core."_
+
+### Avoid gerunds:
+
+Avoid using gerunds (verbs ending in -ing) to make headings and sentences more direct.
+
+- ✅ _"Run pipelines to produce results."_
+- ❌ _"Running pipelines produces results."_
+
+### Be direct and brief
+
+Be direct and brief in your instructions and descriptions. Avoid words and qualifiers like _may_, _might_, _should_, _could_, _just_, or _even_.
+
+- ✅ _"To make changes to the pipeline"_
+- ❌ _"If you would like to make changes to the pipeline"_
+- ✅ _"Install with Conda"_
+- ❌ _"You can also install with Conda"_
+
+### Avoid Latin abbreviations (e.g., i.e., etc.)
+
+Write out the full phrase instead of using Latin abbreviations.
+
+- ✅ _"For example,"_ not ❌ _"e.g."_
+- ✅ _"That is,"_ not ❌ _"i.e."_
+
+### Avoid please and thank you
+
+Maintain a professional and direct tone by avoiding the use of ❌ _"please"_ and ❌ _"thank you"_.
+
+## Formatting guidelines
+
+:::tip
+See [Markdown on the nf-core website](/docs/contributing/website/markdown#checklist) for instructions
+on how to use markdown formatting and special elements such as "admonitions" for nice formatting.
+You're reading an admonition right now!
+:::
+
+### Code and commands
+
+Use inline code formatting for commands, file names, and code snippets.
```

Comment:
```suggestion
Use inline code formatting for short commands, file names, and code snippets.
```

I 've seen multi-line in-line code formatting blocks...

Reply:
code snippets should imo be code blocks not inline code, because code blocks get the "copy-code" button

Reply:
Yeah they're not mentioned in this page at all. Needs an extra bullet / point for them.